### DISCUSSION TITLE
Non-QWERTY User Needs to Enable Arrow Keys Everywhere

### DISCUSSION THREAD

**Discussion by animanna** [m4xshen/hardtime.nvim#124] [Q&A] [answered]
Hi I really found this plugin valuable to enhance my nvim workflow as a guide and a very graceful form of kinesthetic learning. But I am having issues with the Arrow Keys. I need to enable arrow keys everywhere as the 'j,k,l' keys are far from my home row, except the h key. By the way I use two layouts to write in different languages so translating qwerty to my layouts (Dvorak and Turkish F) had never been the option for me. 

So I don't use `h j k l` or home row motions in nvim and I don't want to do a transmap as things are useful enough as they are. Even switching between two layouts for different languages and restructuring motions is quite taxing on my neuroplasticity. Turkish F can be a nightmare with imposed hjkl, they go all over the place randomly, which is an epic flow breaker.

**Comment by animanna** [comment, marked as answer]
I tinkered with it and could solve my problem with this basic config. But I would be happy to here if the advanced users have any tips about this situation.

```
return {
  "m4xshen/hardtime.nvim",
  dependencies = { "MunifTanjim/nui.nvim", "nvim-lua/plenary.nvim" },
  opts = {
    disabled_keys = {
      ["<Up>"] = {},
      ["<Down>"] = {},
      ["<Left>"] = {},
      ["<Right>"] = {},
    },
  },
}
```

**Reply by m4xshen** [reply]
Thanks for the feedback! I think this is the ideal way to enable arrow keys in every mode.
